mkdd_editor.py

```diff
@@ -2787,6 +2790,27 @@ def action_update_position(self, event, pos):
 
         self.statusbar.showMessage(display_string)
 
+    def dragEnterEvent(self, event):
+        mime_data = event.mimeData()
+        if mime_data.hasUrls:
```

Comment:
Note that the `()` are missing, and therefore `hasUrls()` is not being invoked. This is only testing whether the member is resolved to `True` (which will always be).